### PR TITLE
Blog post about managing slack channels

### PR DESCRIPTION
**Pull request by haacked**
I plan to add a screenshot showing it in action once we publish the actual feature.

### CODE REVIEW

**Review by pmn — APPROVED** [review]
LGTM!

File: _posts/2021/2021-11-17-manage-slack-channels-with-abbot.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Abbot skill authors now have access to the Rooms API (via `bot.rooms`) to manage channels in Slack. This API lets skill authors create new channels, invite users to a channel, set a topic or purpose for a channel, and archive a channel. This feature can come in handy when building out your own incident workflow, especially when combined with [signals](/archive/2021/11/10/introducing-signals/)!
+
+This feature requires that you grant a couple of additional Slack scopes to Abbot, `channels:manage` and `groups:write`. When you log in to [Abbot](https://ab.bot/) with your Slack account, you'll be prompted to grant these scopes in the header:
+
+![Image with header that says "Abbot has new capabilities! These capabilities require updated permissions from Slack. Please reinstall Abbot for the best experience."](https://user-images.githubusercontent.com/19977/142256140-13e09b8d-c475-4baa-ace5-96d490caf426.png "Reinstallation is quick and painless")
+
+To help with learning how this feature works, we published a set of demonstration skills in each language:
+
+* [`room-cs` - C#](https://github.com/aseriousbiz/abbot-skills/blob/main/room-cs/main.csx)
```
Comment (shana) [inline _posts/2021/2021-11-17-manage-slack-channels-with-abbot.md:18]: The skill has the `#load ".meta/globals.csx" // This is required for Intellisense in VS Code, etc. DO NOT TOUCH THIS LINE!` line on top, which won't compile if people copy-paste it directly into the skill editor. Given that the message says "DO NOT TOUCH", that's going to be a bit confusing...

Comment (haacked) [reply _posts/2021/2021-11-17-manage-slack-channels-with-abbot.md:18]: Good call, I fixed it up.

**Review by shana — APPROVED** [review]
🚀

**Pull request merged**